### ISSUE TITLE
User.java:97-98: Refactor so that this returns a...

### ISSUE THREAD

**Issue by 0pdd** [jcabi/jcabi-github#1400]
The puzzle `1304-bb172082` from #1304 has to be resolved:

https://github.com/jcabi/jcabi-github/blob/5ded5a2945840694fc106994de7a374ee8968c8c/src/main/java/com/jcabi/github/User.java#L97-L98

The puzzle was created by Mihai A on 25-Apr-18. 

Estimate: 30 minutes, role: DEV.

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@amihaiemil[/z](https://www.0crat.com/u/amihaiemil) please, pay attention to this issue

**Comment by 0crat** [comment]
@0pdd[/z](https://www.0crat.com/u/0pdd) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C9R53K5JA) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by 0crat** [comment]
Job [#1400](https://github.com/jcabi/jcabi-github/issues/1400) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#1400](https://github.com/jcabi/jcabi-github/issues/1400) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/C9R53K5JA/1150821099); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by llorllale** [comment]
@0crat waiting

**Comment by 0crat** [comment]
> @0crat waiting  ([here](https://github.com/jcabi/jcabi-github/issues/1400#issuecomment-386834399))

@llorllale The impediment for [#1400](https://github.com/jcabi/jcabi-github/issues/1400) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by llorllale** [comment]
@0crat waiting on review for #1405

**Comment by 0crat** [comment]
> @0crat waiting on review for #1405 ([here](https://github.com/jcabi/jcabi-github/issues/1400#issuecomment-386875960))

@llorllale Job [#1400](https://github.com/jcabi/jcabi-github/issues/1400) is already on hold

**Comment by 0pdd** [comment]
The puzzle `1304-bb172082` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#1406](https://github.com/jcabi/jcabi-github/issues/1406), [#1407](https://github.com/jcabi/jcabi-github/issues/1407) are still not solved.

**Comment by 0crat** [comment]
Order was finished: +30 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The job [#1400](https://github.com/jcabi/jcabi-github/issues/1400) is now out of scope

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1407](https://github.com/jcabi/jcabi-github/issues/1407) is still not solved; solved: [#1406](https://github.com/jcabi/jcabi-github/issues/1406).

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#1417](https://github.com/jcabi/jcabi-github/issues/1417), [#1418](https://github.com/jcabi/jcabi-github/issues/1418) are still not solved; solved: [#1406](https://github.com/jcabi/jcabi-github/issues/1406), [#1407](https://github.com/jcabi/jcabi-github/issues/1407).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1418](https://github.com/jcabi/jcabi-github/issues/1418) is still not solved; solved: [#1406](https://github.com/jcabi/jcabi-github/issues/1406), [#1407](https://github.com/jcabi/jcabi-github/issues/1407), [#1417](https://github.com/jcabi/jcabi-github/issues/1417).

**Comment by 0pdd** [comment]
@0pdd all 4 puzzles are solved here: [#1406](https://github.com/jcabi/jcabi-github/issues/1406), [#1407](https://github.com/jcabi/jcabi-github/issues/1407), [#1417](https://github.com/jcabi/jcabi-github/issues/1417), [#1418](https://github.com/jcabi/jcabi-github/issues/1418).
